dashboard/src/components/localhost/AttendeeRequestList.vue

```diff
@@ -195,6 +213,7 @@ import {
 } from 'frappe-ui'
 import { ref } from 'vue'
 import RequestDetailDialog from '@/components/localhost/RequestDetailDialog.vue'
+import { truncateStr } from '../helpers/utils'
```

Comment:
`@/components/helpers/utils` might be better?

Reply:
The helper dir shouldn't be inside components - I have updated the path